server/route.go

```diff
@@ -485,6 +485,12 @@ func (c *client) processRouteInfo(info *Info) {
 		if !s.getOpts().Cluster.NoAdvertise {
 			s.addClientConnectURLsAndSendINFOToClients(info.ClientConnectURLs)
 		}
+
+		// This will allow us to determine the initial RTT without having to
+		// wait for first timer based PING.
+		c.mu.Lock()
+		c.sendPing()
```

Comment:
This work for both sides?

Reply:
Yes, this code (createRoute) is invoked in both cases: when soliciting and when accepting. The two tests I added (monitoring and regular) check that RTT is updated both on the initiating and accepting side.

Reply:
ok thx. LGTM